nipype/pipeline/plugins/legacymultiproc.py

```diff
@@ -184,6 +184,8 @@ def __init__(self, plugin_args=None):
         non_daemon = self.plugin_args.get('non_daemon', True)
         maxtasks = self.plugin_args.get('maxtasksperchild', 10)
         self.processors = self.plugin_args.get('n_procs', cpu_count())
+        if self.plugin_args.get('max_jobs', np.inf) < self.processors:
```

Comment:
Maybe just:
```suggestion
        if self.plugin_args.get('max_jobs) is not None:
```

Reply:
in some cases, this may already be set?https://github.com/nipy/nipype/blob/232ac23bfd48240906e7e629e98eec854bfdcb1c/nipype/pipeline/plugins/base.py#L104